HelpSource/Classes/ScopeView.schelp

```diff
@@ -131,63 +147,61 @@ This explains all the options:
 code::
```

Comment:
It looks like the example is indented 8 spaces wide because you have 2 levels of indentation throughout. Try and make the example match the code snippet bellow:
```
(
s.waitForBoot({
    var func, sdef1, sdef2, syn1, syn2, startButton;
    f = Buffer.alloc(s,1024,2);
    b = Bus.audio(s,1);

    w = Window("Scope", Rect(150, Window.screenBounds.height-500,790,400)).front;
    c = ScopeView(w,Rect(10,10,380,380)); // this is SCScope
    c.bufnum = f.bufnum;

    // IMPORTANT
    c.server = s;
...etc
```

Reply:
Looks good!